sentry-opentelemetry/sentry-opentelemetry-core/src/main/java/io/sentry/opentelemetry/SpanDescription.java

```diff
@@ -0,0 +1,34 @@
+package io.sentry.opentelemetry;
+
+import io.sentry.protocol.TransactionNameSource;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+
+@ApiStatus.Internal
+public final class SpanDescription {
```

Comment:
m: Maybe name this `OtelSpanInfo` or similar to make it more clear about how this was created.

`spanDescription.getDescription()` seems confusing to me.

Reply:
Will rename in #2380 if that's OK for you?

Reply:
Yup that works!